pull_dataset.py

```diff
@@ -0,0 +1,38 @@
+# This script is only used by the CI Pipeline
+import logging
+import os
+
+import click
+from jinacld_tools.aws.services.s3 import S3
+
+
+BUCKET_NAME = "kaggle-data-set-bucket"
```

Comment:
@jacobowitz Can you send be the name of the bucket we want to use for storing the data-sets, then I will add it here

Reply:
sure, lets use _jina-examples-datasets_

Reply:
Thanks, https://github.com/jina-ai/examples/pull/538/commits/598f7cb76f3a7254595e121b8e05ea31af0db814